preloading-callbacks/src/main/java/settingdust/preloadingtricks/util/ServiceLoaderUtil.java

```diff
@@ -22,15 +23,21 @@ public static <T> void loadServices(Class<T> clazz, ServiceLoader<T> serviceLoad
         } while (!hasNext);
         while (hasNext) {
             final var provider = iterator.next();
+
+            String providerName = provider.type().getName();
+
+            if (!providerName.startsWith("settingdust.preloadingtricks.")) {
+                logger.info("Preloading-tricks is loading " + provider.type().getName());
+            }
+
             try {
-                logger.info("Loading " + provider.type().getName());
                 provider.get();
             } catch (Throwable t) {
-                logger.error(
-                        "Loading {} failed. Stacktrace is in DEBUG level. {}",
-                        provider.type().getName(),
-                        t.getMessage());
-                logger.debug("Loading " + provider.type().getName() + " failed.", t);
+
+                if (!providerName.startsWith("settingdust.preloadingtricks.")) {
+                    logger.debug("Loading " + provider.type().getName() + " failed.");
```

Comment:
I think we still need a error/warn level log for developer to notice if it is not expected

Reply:
yes